core/src/main/java/org/apache/spark/memory/MemoryConsumer.java

```diff
@@ -41,7 +41,7 @@ protected MemoryConsumer(TaskMemoryManager taskMemoryManager, long pageSize, Mem
   }
 
   protected MemoryConsumer(TaskMemoryManager taskMemoryManager) {
```

Comment:
yea, but this PR fixed it, so after this PR, the memory mode is always same with `TaskMemoryManager`'s memory mode, right?